### ISSUE TITLE
[BUG] `stdout` not in proper JSON format  when `--format json` is used

### ISSUE THREAD

**Issue by mitchspano** [forcedotcom/sfdx-scanner#771]
**Describe the bug**
The output of the `scanner:run` command is not in JSON format when `--format json` is used.

**To Reproduce**
Execute the below command
```shell
$ sfdx scanner:run --format json --target path/to/some/directory
```
The `stdout` will contain a header with a call to action to provide feedback, and the JSON will be printed below the header:
```
=== We're constantly improving Salesforce Code Analyzer. Tell us what you think! Give feedback at https://research.net/r/SalesforceCA.

[{"engine":"pmd","fileName":"/path/to/some/directory/MyClass.cls"
...
}]
```
This prevents us from parsing the `stdout` directly to an object.

**Expected behavior**
We expect that the `stdout` from executing that command would be in JSON format so that it can be parsed into an object.

**Desktop (please complete the following information):**
 - OS: Mac OS Monterey 12.4
 - Scanner Version 2.13.6

**Workaround**
Split the `stdout` on `\n` character or take the substring beginning at the first `[` character.

**Urgency**
Not business-stopping.

**Comment by rmohan20** [comment]
@mitchspano Thanks for alerting us about this issue and sharing the workaround as well. We'll keep you updated on our fix.

**Comment by rmohan20** [comment]
@mitchspano Another workaround would be to use `--outfile` flag instead of `--format` and use the contents of the output file for further processing. SFCA would guess the format based on the extension of the output file.

Something like this:
```
$ sfdx scanner:run --target path/to/some/directory --outfile path/to/your-file.json
```

In the meantime, we are getting our fix ready and will update here when the changes are published.

**Comment by rmohan20** [comment]
FYI, the latest 2.x version, `2.13.7`, fixes this issue.
A fix is ready for 3.x version and will be released in a couple of weeks. Leaving this issue open until then.

@mitchspano Please do let us know if the issue still persists. And thanks again for reporting the problem.

**Comment by rmohan20** [comment]
We've fixed this issue in the latest version 3.4.0 as well. Thanks for reporting.
